assets/i18n/qqq.json

```diff
@@ -105,19 +105,19 @@
 	"I18N_CLASSROOM_CALLOUT_HEADING_1": "The header displayed in the callout to the classroom page which is present in the library page.",
 	"I18N_CLASSROOM_CALLOUT_HEADING_2": "The subheading for the callout to the classroom page which is present in the library page.",
 	"I18N_CLASSROOM_CALLOUT_PARAGRAPH_1": "Text for description of the callout to the classroom page which is present in the library page.",
-	"I18N_CLASSROOM_MATH_TITLE": "The text that is displayed on the classroom page - Text shown as classroom name.",
+	"I18N_CLASSROOM_MATH_TITLE": "The classroom name that is displayed on the classroom page.",
 	"I18N_CLASSROOM_PAGE_COMING_SOON": "The text displayed on a topic card in the classroom page when it is not published and inaccessible to the learner.",
 	"I18N_CLASSROOM_PAGE_COURSE_DETAILS": "The header displayed above the course details section in the classroom page.",
 	"I18N_CLASSROOM_PAGE_HEADING": "The header displayed above the classroom name in the classroom page.",
 	"I18N_CLASSROOM_PAGE_SEARCH_BAR_HEADING": "The heading for the search bar at the bottom of the classroom page.",
 	"I18N_CLASSROOM_PAGE_SEARCH_BAR_SUBHEADING": "The subheading for the search bar at the bottom of the classroom page.",
 	"I18N_CLASSROOM_PAGE_TOPICS_COVERED": "The header displayed above the topics list section in the classroom page.",
-	"I18N_COLLECTION_PLAYER_PAGE_BEGIN": "Part of title text displayed on classroom page - Text is shown when explorations are not started in a collection.",
-	"I18N_COLLECTION_PLAYER_PAGE_CONTINUE": "Part of title text displayed on classroom page - Text is shown when explorations have been started in a collection.",
-	"I18N_COLLECTION_PLAYER_PAGE_FINISHED": "The title text displayed on classroom page - Text is shown when all explorations have been finished by the user.",
-	"I18N_COLLECTION_PLAYER_PAGE_HOVER_MESSAGE": "Message shown on preview card on classroom page.",
-	"I18N_COLLECTION_PLAYER_PAGE_NO_EXPLORATION": "The title text displayed on classroom page - Text is shown when no explorations are present in the collection.",
-	"I18N_COMMUNITY_LIBRARY_PAGE_COLLECTION": "The text displayed on the community library page collection cards - Text is shown if card is that of the collection.",
+	"I18N_COLLECTION_PLAYER_PAGE_BEGIN": "Part of collection title text displayed on collection player page - It is shown when explorations are not started in a collection.",
```

Comment:
I still don't know what "title text" means.

If you search for "title text" it means "the text of images which a user sees after hovering over the image" -- is this what you are implying? If not, perhaps you mean "heading" or similar.

Reply:
Oh, yes, it's actually a heading. Will update it. Thanks!

Reply:
Oh, yes, it's actually a heading. Will update it. Thanks!

Reply:
Done!